SharpBucket/V2/EndPoints/PullRequestsResource.cs

```diff
@@ -131,6 +131,11 @@ internal Comment GetPullRequestComment(int pullRequestId, int commentId)
             return _repositoriesEndPoint.GetPullRequestComment(_accountName, _repository, pullRequestId, commentId);
         }
 
+        internal Comment PostPullRequstComment(int pullRequestId, Comment comment)
```

Comment:
Same typo issue here
```suggestion
        internal Comment PostPullRequestComment(int pullRequestId, Comment comment)
```